sky/backends/backend_utils.py

```diff
@@ -882,6 +882,10 @@ def write_cluster_config(
                 'dump_port_command': dump_port_command,
                 # Ray version.
                 'ray_version': constants.SKY_REMOTE_RAY_VERSION,
+                # Command for waiting ray cluster to be ready on head.
+                'ray_head_wait_initalized_command':
+                    instance_setup.RAY_HEAD_WAIT_INITALIZED_COMMAND,
```

Comment:
```suggestion
                'ray_head_wait_initialized_command':
                    instance_setup.RAY_HEAD_WAIT_INITIALIZED_COMMAND,
```
typo?

Reply:
Good catch! Replaced all the occurance. Thanks!